geomesa-compute/src/main/scala/org/locationtech/geomesa/compute/spark/GeoMesaSpark.scala

```diff
@@ -186,27 +192,20 @@ class GeoMesaSparkKryoRegistrator extends KryoRegistrator {
 
   override def registerClasses(kryo: Kryo): Unit = {
     val serializer = new com.esotericsoftware.kryo.Serializer[SimpleFeature]() {
-      val typeCache = CacheBuilder.newBuilder().build(
-        new CacheLoader[String, SimpleFeatureType] {
-          override def load(key: String): SimpleFeatureType = {
-            val spec = System.getProperty(GeoMesaSpark.typeProp(key))
```

Comment:
Quite literally with `sc.broadcast(sfts)` where `sfts` is a map of type name to string encoding. Each partition then grabs this, creates the type, and puts it in their cache